app/models/project.rb

```diff
@@ -682,8 +703,6 @@ def task_completion_csv
       portfolio_status,
       grade > 0 ? grade : '',
       grade_rationale,
-      tutorial ? tutorial.abbreviation : '',
-      main_tutor.name
```

Comment:
@macite Any potential issues?

Reply:
Yes we will need the list of tutorials for each stream here.

Also need to adjust the header to include the stream abbreviations for each of these new columns.
Also need one column if there are no streams